src/overlays/ovl_i2/9BD00.c

```diff
@@ -233,9 +233,15 @@ void func_i2_80100DDC(u8* arg0, s32 arg1, u16 arg2);
 
 #pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i2/9BD00/func_i2_801034F8.s")
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i2/9BD00/func_i2_8010356C.s")
+s32 func_i2_801034F8(s32, s32);
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i2/9BD00/func_i2_8010358C.s")
+void func_i2_8010356C(s32 arg0) {
+    func_i2_801034F8(arg0, 0xE);
+}
+
+void func_i2_8010358C(s32 arg0) {
+    func_i2_801034F8(arg0 + 0x12, 0xE);
```

Comment:
This looks like it's going to be &arg0->unk_12 but not sure if it's possible to figure out the struct from here

Reply:
Here's a function that uses this one https://decomp.me/scratch/qz4tm
It looks like it's passing in `extern u8 D_i2_8010D7F0[];`

Reply:
ah alright, I think that D_i2_8010D7F0 is eventually gonna be an array of a struct of size 0x19E0 but I haven't quite figured out what yet so I'm happy to leave this as is, thanks for looking into it!